src/main/scala/mesosphere/mesos/TaskGroupBuilder.scala

```diff
@@ -220,23 +224,25 @@ object TaskGroupBuilder extends StrictLogging {
       val containerInfo = mesos.ContainerInfo.newBuilder
         .setType(mesos.ContainerInfo.Type.MESOS)
 
-      mesosNetworks.foreach(containerInfo.addNetworkInfos(_))
+      mesosNetworks.foreach(containerInfo.addNetworkInfos)
 
-      podDefinition.volumes.collect{
+      podDefinition.volumes.collect {
         // see the related code in computeContainerInfo
         case e: EphemeralVolume =>
           mesos.Volume.newBuilder()
+            .setContainerPath(ephemeralVolumePathPrefix + e.name.getOrElse(""))
```

Comment:
what is this semantically if we fall back to default `""` ?

Reply:
Adjusted.